event_attribution_reporting_clicks.md

```diff
@@ -103,18 +103,21 @@ Attribution Source Declaration
 An attribution source is an anchor tag with special attributes:
 
 `<a attributiondestination="[eTLD+1]" attributionsourceeventid="[64-bit unsigned integer]"
-attributionexpiry=[unsigned long long] attributionreportto="[origin]">`
+attributionexpiry=[unsigned long long] attributionreportto="[origin]"
+attributionsourcepriority="[64-bit unsigned integer"]>`
```

Comment:
Priority is signed https://wicg.github.io/conversion-measurement-api/#dom-htmlanchorelement-attributionsourcepriority